contributors/devel/flexvolume.md

```diff
@@ -1,6 +1,6 @@
 # Flexvolume
 
-Flexvolume enables users to write their own drivers and add support for their volumes in Kubernetes. Vendor drivers should be installed in the volume plugin path on every Kubelet node and on master node(s) if `--enable-controller-attach-detach` Kubelet option is enabled.
+Flexvolume enables users to write their own drivers and add support for their volumes in Kubernetes. Vendor drivers should be installed in the volume plugin path on every node, and on master if the driver requires attach capability.
```

Comment:
This statement is not completely true. If the --enable-controller-attach-detach is false, kubelet will do the attach.

Reply:
I think we should strongly discourage users from setting that flag to false; k8s is no longer designed to operate in that mode. Having this flag in the documentation gives users new to k8s the idea that disabling attach detach is a common and viable alternative, but in most cases it's not.

I'm open to wording it differently but I'd avoid including the flag in the documentation